cors/cors.go

```diff
@@ -0,0 +1,150 @@
+package cors
+
+import (
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+)
+
+const accessControlMaxAge = "600" //10 minutes
+
+type OriginChecker interface {
+	Options(events.APIGatewayV2HTTPRequest) events.APIGatewayV2HTTPResponse
+	GetCorsHeaders(request events.APIGatewayV2HTTPRequest) map[string]string
+}
+
+// Domains contains the allowed domains and the accepted methods for each one
+type Domains struct {
+	Allowed map[string]map[string]bool
+}
+
+// NewOriginChecker returns the default Domains
+func NewOriginChecker() OriginChecker {
+	acceptedDomains := map[string]map[string]bool{
+		"thelist.app":    {http.MethodDelete: true, http.MethodGet: true, http.MethodPatch: true, http.MethodPost: true},
+		"localhost:3000": {http.MethodDelete: true, http.MethodGet: true, http.MethodPatch: true, http.MethodPost: true},
```

Comment:
Oh, one more thing, can we add the dev domain:
```suggestion
		"thelist.app":    {http.MethodDelete: true, http.MethodGet: true, http.MethodPatch: true, http.MethodPost: true},
		"dev.thelist.app": {http.MethodDelete: true, http.MethodGet: true, http.MethodPatch: true, http.MethodPost: true},
		"localhost:3000": {http.MethodDelete: true, http.MethodGet: true, http.MethodPatch: true, http.MethodPost: true},
```